testfiles/5ZoneCAV_MaxTemp_JSON_Outputs.idf

```diff
@@ -3628,3 +3628,5 @@
   Output:Table:SummaryReports,
     AllSummary;              !- Report 1 Name
 
+  Output:SQLite,
+    SimpleAndTabular;        !- Option Type
```

Comment:
I found zero file that already had three conditions:
* Output:Table:SummaryReports,AllSummary
* Output:JSON with TimeseriesAndTabular and Output json set to Yes 
* Output:SQLite with SimpleAndTabular

So I added SQLite to this one.

Reply:
Great reuse of an existing file, thanks!